src/types/message.ts

```diff
@@ -136,3 +139,17 @@ export type SignToPageMessage = ExtrinsicSignResponse;
 export type ToPageMessages = BgToPageMessage | SignToPageMessage;
 
 export type ToBgMessage = InitCommand | SignCommand;
+
+export type ToContentMessage = InitCommand;
+
+export enum EpicMessageOrigin {
+  PAGE = 'redirect:page',
```

Comment:
it's more like `forward` than `redirect`
or from epic's view, you can say `from:xxx`